include/api.php

```diff
@@ -73,31 +73,51 @@ function pll_default_language( $field = 'slug' ) {
 }
 
 /**
- * Among the post and its translations, returns the id of the post which is in the language represented by $lang.
+ * Among the post and its translations, returns the ID of the post which is in the language represented by $lang.
  *
  * @api
  * @since 0.5
+ * @since 3.4 Returns 0 instead of false.
+ * @since 3.4 $lang accepts PLL_Language, string, or int.
```

Comment:
OK for `PLL_Language` but I don't agree for `int`.

Reply:
Done in [10ec660](https://github.com/polylang/polylang/pull/1149/commits/10ec660bc67a078dde283dd1c86b5566cee60343).